core/src/migrations/000002-createDestinationGroups.ts

```diff
@@ -1,47 +1,48 @@
-export default {
-  up: async function (migration, DataTypes) {
-    await migration.sequelize.transaction(async () => {
-      await migration.createTable("destinationGroups", {
-        guid: {
-          type: DataTypes.STRING(40),
-          primaryKey: true,
-        },
+import Sequelzie from "sequelize";
```

Comment:
```suggestion
import Sequelize from "sequelize";
```

looks like the typo is in all migrations :/